configure.ml

```diff
@@ -276,6 +276,7 @@ module Prefs = struct
   let flambda_flags = ref []
   let debug = ref true
   let profile = ref false
+  let bin_annotate = ref false
```

Comment:
This is nitpicking but I would have named the two options `-bin-annot` and `-annot` to reflect exactly the underlying compiler options. If you had used these two names, you could have made `-annotate` emit a warning saying: no op, use `-bin-annot` or `-annot` instead.

Reply:
I like @Zimmi48 suggestion and implemented it.